chapters/annotations.tex

```diff
@@ -447,7 +447,8 @@ \section{Simulations}\label{annotations-for-simulations}\label{simulations}
 \end{annotationdefinition}
 
 
-\section{Single Use of Class}\label{annotation-for-single-use-of-class}\label{single-use-of-class}
+\section{Usage Restrictions}\label{usage-restrictions}
```

Comment:
I would have have applied the table-based style of presentation to a new section like this, but  it might be better to include such as change in #2957…

Reply:
I think it is best to wait, especially as I separated the two annotations.